tests/schema_registry/test_avro_serializer.py

```diff
@@ -111,7 +111,7 @@ def test_avro_serializer_record_subject_name_strategy(load_avsc):
                                      conf={'subject.name.strategy':
                                            record_subject_name_strategy})
 
-    ctx = SerializationContext('test_subj', MessageField.VALUE)
+    ctx = SerializationContext('test_subj', MessageField.VALUE, [])
```

Comment:
From a test perspective we yield better coverage if some of these use the default value (not specified), and some use an empty list